### ISSUE TITLE
Polyfill IntersectionObserver is not served in browser Safari

### ISSUE THREAD

**Issue by shYkiSto** [polyfillpolyfill/polyfill-service#1851]
Polyfill IntersectionObserver is not served in browser Safari. According to https://caniuse.com/#feat=intersectionobserver it should be

```curl 'https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.includes,Array.prototype.find,Array.prototype.findIndex,Array.prototype.keys,Array.isArray,Object.entries,Object.values,IntersectionObserver,IntersectionObserverEntry,requestAnimationFrame,fetch' -XGET -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0.2 Safari/605.1.15'```

**Comment by kelvinlouis** [comment]
It is also missing on IE11. `fetch` isn't provided either. We are using the following URL: `https://cdn.polyfill.io/v2/polyfill.js?features=default-3.6,WeakMap,fetch,Array.prototype.find,Array.prototype.includes,IntersectionObserver,Element.prototype.dataset,HTMLPictureElement`

**Comment by shYkiSto** [comment]
I also noticed Object.values being missed for IE. This started happening right after the outage. And IntersectionObserver issue appears to be ~1 month old

**Comment by JakeChampion** [comment]
These should be fixed as part of the fix for #1853

**Comment by davidenglishmusic** [comment]
There may have been a regression since. It doesn't appear to be getting served to Safari currently:

<img width="1280" alt="screen shot 2019-03-07 at 10 43 37 am" src="https://user-images.githubusercontent.com/4421670/53969007-0601e300-40c6-11e9-8cbf-aa04464f691d.png">

**Comment by gpoole** [comment]
I'm also experiencing this issue with IE11.

For the following config: https://polyfill.io/v3/polyfill.js?features=IntersectionObserver-polyfill%2CIntersectionObserverEntry-polyfill

The response to IE11 is:

```js
/* Polyfill service v3.31.1
 * For detailed credits and licence information see https://github.com/financial-times/polyfill-service.
 * 
 * Features requested: IntersectionObserver-polyfill,IntersectionObserverEntry-polyfill
 *  */

/* No polyfills found for current settings */
```

![Screen Shot 2019-03-12 at 12 02 14 pm](https://user-images.githubusercontent.com/2898433/54167620-c6b90500-44be-11e9-839b-f6cf44309fd6.png)

**Comment by gpoole** [comment]
Actually my mistake, my problem was not the same issue, it looks like the generated URL was incorrectly including `-polyfill` in the feature requests. It should have been:
https://polyfill.io/v3/polyfill.js?features=IntersectionObserver%2CIntersectionObserverEntry, which does work correctly in IE11. Seems to have been fixed in #1906.
